apps/generator/test/integration.test.js

```diff
@@ -3,6 +3,8 @@
  */
 
 const { readFile } = require('fs').promises;
+const { promises: fsPromise } = require('fs');
+const { readFileSync} = require('fs');
```

Comment:
please don't do it this way, in line 5 you already have a require where readFile in promise version is included, you can extend it with `mkdir` and others if needed, like `const { mkdir, writeFile, readFile } = require('fs').promises;`